rust/stackable-cockpit/src/xfer/cache.rs

```diff
@@ -291,6 +293,21 @@ pub enum CacheBackend {
     Disabled,
 }
 
+pub enum DeleteFilter {
+    All,
+    OnlyExpired,
+}
+
+impl From<bool> for DeleteFilter {
+    fn from(value: bool) -> Self {
+        if value {
+            Self::OnlyExpired
+        } else {
+            Self::All
+        }
+    }
+}
```

Comment:
Please be judicious about when you implement `From<T>`. For `bool` especially, you'll often need a lot of additional context to understand what the field actually means. Instead, either introduce a separate named associated function, or just inline the conversion to the one place where it's actually used.

Reply:
Yeah. I had similar thoughts. Will add an `from_bool` method to `DeleteFilter`.

Reply:
Added in f7b0df5

Reply:
What name do you suggest?

Reply:
I'd just inline it, since it's only really useful in the context of the CLI. Alternatively, we could implement `clap::Parser` on `DeleteFilter`, but honestly that seems a bit overkill.

Reply:
> Alternatively, we could implement clap::Parser on DeleteFilter, but honestly that seems a bit overkill.

My thoughts as well. Let's inline it I guess...

Reply:
Inlined in a2a4dc1